CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing guidelines
+
+A single gazetteer is a collection of places that are thematically related or relevant to a particular map styling task. Keeping the content of a gazetteer focused and clearly defined will help map designers pick suitable gazetteers for reviewing their map styling projects.
```

Comment:
Starting with a gazetteer definition is great, however I think stating "... thematically related or relevant to a particular map styling task ..." seems like a specific descriptor for a particular use-case. I would suggest keeping the definition vague, such as "an index of places," and moving "thematic relationships and map styling tasks" to the intention of this particular online gazetteer repository.